sql/signup_and_login/R__check_addr_can_be_added_to_account.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE FUNCTION addr_can_be_added (account_id uuid, addr text, OUT can_be_added boolean)
+AS $$
+DECLARE
+    v_account_id uuid = account_id;
+    v_addr text = addr;
+    associated_account_id uuid;
+BEGIN
+    PERFORM
+        id
+    FROM
+        account
+    WHERE
+        id = v_account_id;
+    IF NOT found THEN
+        RAISE EXCEPTION
+            USING message = 'no account found for given account_id', hint = 'check the account_id';
+    END IF;
+    associated_account_id := public.get_account_by_addr (v_addr);
+    IF associated_account_id IS NULL THEN
+        can_be_added := 'true';
```

Comment:
can't we just set it like this?
```suggestion
        can_be_added := true;
```

Reply:
indeed, either way is an acceptable syntax. i've adjusted it though for consistency (consistency is good)

1da2b541b70d31c3da15448cc513dd330de28af6